### PR TITLE
[CLAS] redirects

### LINKED ISSUE #7761: Import of url redirects for CLAS
# Story

As a site owner of clas.uiowa.edu, I would like our CSV of redirects imported to our new v3 website, so we don't have to manually recreate all of them prior to launch.

# Additional Info

We created an export in #7399 

CLAS has removed entries no longer needed.

We imported redirects for Tippie previously and likely can reuse some of that work.

Grooming conversation https://github.com/uiowa/uiowa/issues/7399#issuecomment-2148358869

It was mentioned that we can probably move forward on some of this prior to having the final csv.

### PR DESCRIPTION
**Pull request by bspeare**
Resolves https://github.com/uiowa/uiowa/issues/7761. 

# How to test

```
ddev blt ds --site=clas.uiowa.edu 
```
1. Test a few redirects to see if they were set.

### CODE REVIEW

File: docroot/sites/clas.uiowa.edu/modules/clas_core/clas_core.install [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * @file
+ * Install tasks.
+ */
+
+use Drupal\redirect\Entity\Redirect;
+
+/**
+ * CLAS redirects.
+ */
+function clas_core_update_8001() {
```
Comment (joewhitsitt) [inline docroot/sites/clas.uiowa.edu/modules/clas_core/clas_core.install]:
```suggestion
function clas_core_update_10001() {
```
we usually align with the version of drupal we are on at the time these are triggered

Comment (bspeare) [reply docroot/sites/clas.uiowa.edu/modules/clas_core/clas_core.install]: Thanks for the tip!

**Comment by joewhitsitt** [comment]
pulling to test quick

**Review by joewhitsitt — APPROVED** [review]
411 in csv, 200 redirects before, 611 after ✅ 

spot checked a few ✅

**Comment by bspeare** [comment]
Thanks @joewhitsitt! Moving this back to draft so we can run it in dev.

**Comment by bspeare** [comment]
CLAS reviewed and approved in dev.  I'm going to merge this in unless there's some reason why this should be in a scheduled release.

**Pull request merged**